data/themes/darktable.css

```diff
@@ -753,47 +755,61 @@ overshoot.right
 #main-histogram .toggle
 {
   border: 1.1px solid alpha(@button_bg, 0.3);
-  min-height: 1em;  /* FIXME: what happens if don't set these? -- what is the source */
   min-width: 1em;
```

Comment:
I realize that this line is unnecessary (as is the `min-height` one you removed). As the "normal" button theme already sets these to 1em.

Reply:
Indeed, you're right. I will remove that line too. Thanks for pointing that!